src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/ReissuanceConsent/HealthCertificateReissuanceConsentViewController.swift

```diff
@@ -13,9 +13,10 @@ class HealthCertificateReissuanceConsentViewController: DynamicTableViewControll
 		restServiceProvider: RestServiceProviding,
 		appConfigProvider: AppConfigurationProviding,
 		cclService: CCLServable,
-		certificates: [HealthCertificate],
+		certificates: [DCCCertificateContainerExtended],
```

Comment:
I don't know in which PR this name was introduced, but it's very generic and I had to look it up first to understand what it does. I think something like `DCCReissuanceCertificateContainer` would be much easier to understand as the struct is quite specific in nature.